README.md

```diff
@@ -28,11 +28,17 @@ place to ask about it might be in [ipfs/apps](https://github.com/ipfs/apps) or
   - [Want to hack on IPFS?](#want-to-hack-on-ipfs)
 - [License](#license)
 
-## Use Cases
+## Platforms
```

Comment:
I'm failing to see how akaska and Beaker Browser can be considered platforms (at least as they are today). As I understand, those are apps and they don't allow people to easily extend/plug into. Is that correct assessment?